src/inmanta/deploy/state.py

```diff
@@ -135,7 +135,7 @@ class AgentStatus(StrEnum):
     STOPPED = enum.auto()
 
 
-class BlockedStatus(StrEnum):
+class Blocked(StrEnum):
```

Comment:
rename 6.

motivation: reduce verbosity + make consistent with field name `ResourceState.blocked`

personal opinion: tiny readability improvement => make the change.